lib/src/element/range_slider.cpp

```diff
@@ -0,0 +1,257 @@
+/*=============================================================================
+   Copyright (c) 2016-2020 Joel de Guzman
+
```

Comment:
At least add your name here. Now I see it is based on my code. Would be nice to share the copyright with you.